global.json

```diff
@@ -1,3 +1,6 @@
 {
-    "projects": ["src", "test/WebSites", "samples"]
+    "projects": [ "src", "test/WebSites", "samples" ],
+    "sdk": {
+        "version": "1.0.0-preview2-003121"
```

Comment:
Should we pin this to 3131 (as also update the KoreBuild version)? That's the LTS build of the SDK.